sys/targets/targets.go

```diff
@@ -181,10 +181,13 @@ var List = map[string]map[string]*Target{
 	},
 	"netbsd": {
 		"amd64": {
-			PtrSize:     8,
-			PageSize:    4 << 10,
-			CFlags:      []string{"-m64"},
-			CrossCFlags: []string{"-m64", "-static"},
+			PtrSize:  8,
+			PageSize: 4 << 10,
+			CFlags:   []string{"-m64"},
+			CrossCFlags: []string{"-m64", "-static",
+				"--sysroot", os.ExpandEnv("${SOURCEDIR}/../dest/"),
+			},
+			CCompiler: os.ExpandEnv("${SOURCEDIR}/../tools/bin/x86_64--netbsd-g++"),
```

Comment:
Is this double `-` correct?

Reply:
Yea, It works for me.
If you check the fuchsia command below its also similar.

Reply:
Oops, sorry I misunderstood the lines you were talking about.
Yes, that's the name of g++ binary generated by ./build.sh tools.